### PR TITLE
Filter and update dependencies

### PR DESCRIPTION
**Pull request by dcow**
Add the ability to update individual dependencies using a command line option or the plugin DSL (#42).

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1 +1 @@
-version=2.2.1-SNAPSHOT
+version=2.3.0-SNAPSHOT
```
Comment (rspieldenner) [inline gradle.properties]: Don't change the version string. We use major.minor to reflect the version of gradle it is tested against. So this is 2.2.x.

I'll put out a 2.3.x using gradle 2.3 this week or next.

Comment (dcow) [inline gradle.properties]: Sure. Should I try to rebase this on the 1.8 branch?

Comment (rspieldenner) [inline gradle.properties]: i'll deal with the merges to different branches.

**Comment by rspieldenner** [comment]
I'll give this a deeper code review over the next 2 days. Looks pretty good though. Excellent tests.

**Comment by dcow** [comment]
re tests: Spock makes me want to write good tests. Blame it (;

**Pull request merged**